dizoo/evogym/entry/walker_ddpg_eval.py

```diff
@@ -0,0 +1,117 @@
+import os
+import gym
+import torch
+from tensorboardX import SummaryWriter
+from easydict import EasyDict
+from functools import partial
+
+from ding.config import compile_config
+from ding.worker import BaseLearner, SampleSerialCollector, InteractionSerialEvaluator, AdvancedReplayBuffer
+from ding.envs import BaseEnvManager
+from ding.envs import get_vec_env_setting, create_env_manager
+from ding.policy import DDPGPolicy
+from ding.utils import set_pkg_seed
+
+walker_ddpg_config = dict(
```

Comment:
why define config here, can we reuse someone?

Reply:
fixed.